source/main/physics/Beam.cpp

```diff
@@ -3390,7 +3390,7 @@ void Actor::ToggleHooks(int group, hook_states mode, int node_number)
     // iterate over all hooks
     for (std::vector<hook_t>::iterator it = ar_hooks.begin(); it != ar_hooks.end(); it++)
     {
-        if (mode == MOUSE_HOOK_TOGGLE && it->hk_hook_node->id != node_number)
+        if (mode == MOUSE_HOOK_TOGGLE && it->hk_hook_node->pos != node_number)

```

Comment:
I think `pos` is correct here.

Reply:
✔️ Yes because value of `node_number` is an index: https://github.com/RigsOfRods/rigs-of-rods/blob/cf08a7dd0b930c7f6da8a7ffb23a5dc689e55aed/source/main/gameplay/SceneMouse.cpp#L176